PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx

```diff
@@ -721,93 +723,88 @@ Bool_t AliConversionPhotonCuts::InitPIDResponse(){
   return kFALSE;
 }
 ///________________________________________________________________________
+Bool_t AliConversionPhotonCuts::InitializeElecDeDxPostCalibration(TString filename) {
+  AliInfo("Entering loading of correction map for post calibration");
+
+  TFile *file = TFile::Open(filename.Data());
+  if(!file){
+    AliError(Form("file for electron dEdx post calibration %s not found",filename.Data()));
+    return kFALSE;
+  }else{
+    AliInfo(Form("found %s ",filename.Data()));
+  }
+
+  for(Int_t i=0;i<fnRBins;i++){
+    fHistoEleMapRecalib[i]  = (TH2S*)file->Get(Form("Ele_R%d_recalib",i));
+    fHistoPosMapRecalib[i]  = (TH2S*)file->Get(Form("Pos_R%d_recalib",i));
+  }
+
+  if (fHistoEleMapRecalib[0] == NULL || fHistoEleMapRecalib[1] == NULL ||
+      fHistoEleMapRecalib[2] == NULL || fHistoEleMapRecalib[3] == NULL  ){
+    AliWarning("Histograms for dedx post calibration not found in %s despite being requested!");
```

Comment:
This must be an AliFatal. The code must really crash if the calibration inputs cannot be found but are requested.